app/views/dashboard/_pending_applications.haml

```diff
@@ -10,5 +10,6 @@
               %li= link_to record.user.proper_name, record
           - else
             No pending applications
-        = link_to 'View saved applications for ' + position.name,
+        - count = position.saved_applications.count
```

Comment:
We discussed in person just now iterating through in a similar manner to pending applications.